src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -4,15 +4,7 @@
 
 {% block toolbar %}
     {% if collector.firewall %}
-        {% if collector.token %}
-            {% set is_authenticated = collector.enabled and collector.authenticated  %}
```

Comment:
This should be kept as it's useful later in the file, no?

Reply:
Yes! I've readded the `is_authenticated` variable. Thanks.